src/ngx_http_lua_util.c

```diff
@@ -1437,9 +1437,6 @@ ngx_http_lua_run_thread(lua_State *L, ngx_http_request_t *r,
 
             ctx->cur_co_ctx = next_coctx;
 
-            ngx_log_error(NGX_LOG_ERR, r->connection->log, 0,
-                          "lua coroutine: %s: %s\n%s", err, msg, trace);
```

Comment:
If we do so, this will be different than what Lua / LuaJIT does. They don't write anything. I mean, OpenResty doesn't log anything when wrapping code with `pcall`. If you want to log errors even when they are raised in a code wrapped with `pcall`, then I would consider having this logged as well, and of course we need to change `pcall` then as well. This would differ from what users' expect, though. My only concern is that I'm not sure if that removed log entry removes error logging also from some case where it shouldn't. That's why I asked do anyone of you know if it is only handling when users' code calls `coroutine.resume` (that's the only place where this logging should be silenced).